applications/train_mlp.py

```diff
@@ -5,6 +5,340 @@
 import shutil
 import os
 import gc
+import sys
+import tqdm
+import optuna
+import pickle
+import warnings
+import numpy as np
+import pandas as pd
+from tensorflow.keras import backend as K
+from argparse import ArgumentParser
+
+from ptype.callbacks import MetricsCallback    
+from ptype.data import load_ptype_uq, preprocess_data
+from mlguess.keras.callbacks import get_callbacks
+from mlguess.keras.models import CategoricalDNN
+from mlguess.pbs import launch_pbs_jobs
+from bridgescaler import save_scaler
+
+
+warnings.filterwarnings("ignore")
+
+
+logger = logging.getLogger(__name__)
+
+
+class Objective(BaseObjective):
```

Comment:
Can you move the Objective class to a module file in the ptype directory and import it rather than having it all in the trainer script?

Reply:
Yes!